staging/src/k8s.io/legacy-cloud-providers/azure/azure_vmss.go

```diff
@@ -132,7 +132,7 @@ func (ss *scaleSet) getVmssVM(nodeName string) (string, string, *compute.Virtual
 
 // GetPowerStatusByNodeName returns the power state of the specified node.
 func (ss *scaleSet) GetPowerStatusByNodeName(name string) (powerState string, err error) {
-	_, _, vm, err := ss.getVmssVM(name)
+	_, _, vm, err := ss.getVmssVM(name, cachedData)
```

Comment:
This one of the reasons  i want to actually keep the cache ttl short *not* long.

@feiskyer FYI

Reply:
agreed, default 15minutes is too long

Reply:
so what's the default TTL now?

Reply:
@andyzhangx The default TTL is still what @feiskyer set with the last PR (15min TTL for VMAS and 10min TTL for VMSS) . But I think we should consider making it shorter now that we are using unsafe read for reconcile calls.

Reply:
so what's your suggested default value?

Reply:
I think we should make it 2min. WDYT?